estuary/config.py

```diff
@@ -22,10 +23,10 @@ class Config(object):
     CORS_ORIGINS = ['http://localhost:4200']
     STORY_MANAGER_SEQUENCE = ['ModuleStoryManager', 'ContainerStoryManager']
     ENABLE_AUTH = False
-    OIDC_INTROSPECT_URL = None
-    OIDC_CLIENT_ID = None
-    OIDC_CLIENT_SECRET = None
-    EMPLOYEE_TYPES = []
+    OIDC_INTROSPECT_URL = None  # type: str
```

Comment:
Could we use the Python 3 syntax please?

https://mypy.readthedocs.io/en/stable/cheat_sheet_py3.html#built-in-types